src/main/scala/com/cloudera/sparkts/models/ARIMA.scala

```diff
@@ -688,7 +688,7 @@ class ARIMAModel(
 
     // copy over last maxLag values, to use in iterateARMA for forward curve
     val forward = new BreezeDenseVector[Double](Array.fill(nFuture + maxLag)(0.0))
-    forward(0 until maxLag) := hist(-maxLag to -1)
+    if(maxLag>0) forward(0 until maxLag) := hist(-maxLag to -1)
```

Comment:
One nitpick: to be consistent with the style guidelines, there should be a space after `if` and the body of the conditional statement should be inside curly braces, e.g.

```
if (condition) {
  do thing
}
```